### ISSUE TITLE
Selected state indicators and icons in the left nav have insufficient contrast against their background on mouse hover

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#859]
The following AI-windows mode(s) have this issue:
- Dark mode
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![6 1](https://user-images.githubusercontent.com/22068434/88343459-4a5f4480-ccf6-11ea-9271-1288c49ac05f.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1375.01. Please regress and close at your convenience. Thanks!

**Comment by iamrafan** [comment]
The icons' color issues against its hover background color are fixed in Canary Release: v1.1.1380.01. I will file a separate issue for the "active thumb" (state) indicator not having sufficient contrast against its adjacent color on hover in High Contrast White mode and High Contrast Black modes
